### PR TITLE
Image zoom feature for workflow image

### PR DESCRIPTION
**Pull request by sagar15795**
I have added image zooming feature to zoom workflow image by clicking on it .

I have added it to both Favourite workflow detail activity and workflow detail activity

### CODE REVIEW

**Comment by sagar15795** [comment]
@therajanmaurya 

Please review it. 
Tell me if there is any suggestion or change .

Thanks

**Comment by ianwdunlop** [comment]
From a functionality point of view it seems to work well. I'm happy to merge it but if you want to wait until @therajanmaurya has reviewed the code then I can wait.

**Comment by therajanmaurya** [comment]
@sagar15795 Remove the redundant call of image URI from Database and add Image URI filed in WorkFlow POJO and pass the Image URI to intent putExtra from workflow details.

This is a good way of doing this and makes app efficient.

do these changes and  let me know. when you will be finished.

**Comment by sagar15795** [comment]
@therajanmaurya 
Thanks for the good suggestion.
I have added your suggestion.

Please review it. 
Tell me if there is any suggestion or change .

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -119,6 +119,11 @@ limitations under the License.
                 android:name="android.support.PARENT_ACTIVITY"
                 android:value="org.apache.taverna.mobile.activities.DashboardMainActivity"/>
         </activity>
+
+        <activity android:name=".ui.imagezoom.ImageZoomActivity"
+                  android:label="Image Zoom"
```
Comment (therajanmaurya) [inline app/src/main/AndroidManifest.xml]: Put this String In strings.xml

File: app/src/main/java/org/apache/taverna/mobile/ui/favouriteworkflowdetail/FavouriteWorkflowDetailFragment.java [hunk 2]
```diff
@@ -50,58 +41,53 @@
...
+import org.apache.taverna.mobile.utils.ConnectionInfo;
+
 import butterknife.BindView;
 import butterknife.ButterKnife;
 import butterknife.OnClick;
 
 public class FavouriteWorkflowDetailFragment extends Fragment
         implements FavouriteWorkflowDetailMvpView {
 
+    private static final String ID = "id";
```
Comment (therajanmaurya) [inline app/src/main/java/org/apache/taverna/mobile/ui/favouriteworkflowdetail/FavouriteWorkflowDetailFragment.java]: Give one line space

File: app/src/main/java/org/apache/taverna/mobile/ui/imagezoom/ImageZoomActivity.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_image_zoom);
+
+        ButterKnife.bind(this);
+
+
+        if (savedInstanceState == null) {
+            getSupportFragmentManager().beginTransaction()
+                    .add(R.id.frame_container
```
Comment (therajanmaurya) [inline app/src/main/java/org/apache/taverna/mobile/ui/imagezoom/ImageZoomActivity.java]: Use Utils ActivityUtils for fragment transaction

File: app/src/main/java/org/apache/taverna/mobile/ui/imagezoom/ImageZoomFragment.java [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+
+public class ImageZoomFragment extends Fragment implements ImageZoomMvpView {
+
+    public static final String JPG_URI = "jpgURI";
+
+    public static final String SVG_URI = "svgURI";
+
+    private static final String SERVER_ERROR = "Sever Error. Please try after sometime";
+
+    private static final String TAG = "ImageZoomFragment";
```
Comment (therajanmaurya) [inline app/src/main/java/org/apache/taverna/mobile/ui/imagezoom/ImageZoomFragment.java]: remove static and use getClass.getSimpleName for getting class name for Log Tag

File: app/src/main/java/org/apache/taverna/mobile/ui/imagezoom/ImageZoomFragment.java [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+
+    @BindView(R.id.ivWorkflowImage)
+    ImageView workflowImage;
+    @BindView(R.id.ivClose)
```
Comment (therajanmaurya) [inline app/src/main/java/org/apache/taverna/mobile/ui/imagezoom/ImageZoomFragment.java]: One Line Space

File: app/src/main/java/org/apache/taverna/mobile/ui/imagezoom/ImageZoomFragment.java [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+    ImageView close;
+
+    PhotoViewAttacher mAttacher;
+
+    private String svgURI;
+
+    private String jpgURI;
+
+    private DataManager dataManager;
```
Comment (therajanmaurya) [inline app/src/main/java/org/apache/taverna/mobile/ui/imagezoom/ImageZoomFragment.java]: Remove unused variables

File: app/src/main/java/org/apache/taverna/mobile/ui/imagezoom/ImageZoomPresenter.java [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+import java.io.InputStream;
+
+import rx.Subscription;
+
+public class ImageZoomPresenter extends BasePresenter<ImageZoomMvpView> {
+
+    public static final String NO_IMAGE_URI = "NO Image Found";
+    public static final String DB_ERROR = "There is some problem. Please try after sometime ";
+    private DataManager mDataManager;
+    private Subscription mSubscriptions;
```
Comment (therajanmaurya) [inline app/src/main/java/org/apache/taverna/mobile/ui/imagezoom/ImageZoomPresenter.java]: Remove these unused variables

**Comment by therajanmaurya** [comment]
@sagar15795 make these changes and let @ianwdunlop know to merge this PR.

**Comment by sagar15795** [comment]
@ianwdunlop  

I have made all changes as suggested by @therajanmaurya .
Please merge it .

Thanks

**Comment by therajanmaurya** [comment]
@ianwdunlop this PR is ready to Merge

**Pull request merged**